src/csharp/SDK/Transformation.cs

```diff
@@ -142,6 +142,129 @@ public void DepthImageToColorCamera(Image depth, Image transformed)
             }
         }
 
+        /// <summary>
+        /// Transforms an Image and an Custom Image from the depth camera perspective to the color camera perspective.
+        /// </summary>
+        /// <param name="depth">Depth image to transform.</param>
+        /// <param name="custom">Custom image to transform.</param>
+        /// <param name="interpolationType">Parameter that controls how pixels in custom_image should be interpolated when transformed to color camera space.</param>
+        /// <param name="invalidCustomValue">Defines the custom image pixel value that should be written to \p transformed_custom_image in case the corresponding.</param>
```

Comment:
>Defines the custom image pixel value that should be written to \p transformed_custom_image in case the corresponding. [](start = 45, length = 117)

Defines the custom image pixel value that should be written to transformedCustom in case the corresponding depth pixel can not be transformed into the color camera space.

Reply:
Fixed! Thanks,